src/cy-reporter.js

```diff
@@ -167,7 +174,7 @@ function CypressJUnit(runner, options) {
         if (t.state == 'failed') failures++;
```

Comment:
Instead of individual counters, should better have the `suite` object declared on top:
```
var suite = {
  name: s.suite.name, 
  tests: s.tests.count, 
  failures: 0, 
  skipped: 0, 
  errors: 0, 
  time: (Date.now() - s.suite.timestamp) / 1000, 
  timestamp: new Date(s.suite.timestamp).toUTCString(), 
  file: s.suite.file
}
...
if (t.state == 'failed') suite.failures++;
```